### PR TITLE
Cleanup, use portier API for door control

### PR DESCRIPTION
**Pull request by user890104**
- Remove references to gosho-ivr
- Use separate backend URLs for Auth and Door control

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,27 +2,24 @@
...
 # pip install -r requirements.txt  # FIXME: pyst2 needs patches
 
-cd /var/lib/asterisk/gosho-ivr/sounds/; ./generate-messages.sh
+cd /var/lib/asterisk/initlab-telephony/sounds/; ./generate-messages.sh
 
 cp door_ivr/door_ivr.example.conf door_ivr/door_ivr.conf
 # edit door_ivr/door_ivr.conf
 
-for x in /var/lib/asterisk/gosho-ivr/sounds/files/*; do ln -s "$x" /usr/share/asterisk/sounds/en_US_f_Allison/; done
+for x in /var/lib/asterisk/initlab-telephony/sounds/files/*; do ln -s "$x" /usr/share/asterisk/sounds/en_US_f_Allison/; done
```
Comment (tochev) [inline README.md:22]: @user890104 , keep in mind that if you want to move the folder you will need to:
- move the current solution
- move the virtualenv (`.venv`)
- change the asterisk conf

File: README.md [hunk 2]
```diff
@@ -2,27 +2,24 @@
 
 Repo containing the initLab's IVR telephony system, which enables opening of doors.
```
Comment (tochev) [inline README.md:3]: Next time can we have a normal branch name - `20230904-fixes` is definitely not informative.

File: door_ivr/door_ivr.test.conf [hunk 3]
```diff
@@ -1,5 +1,6 @@
 [backend]
-api_url=http://127.0.0.1:3000/api
+auth_api_url=http://127.0.0.1:3000/api
+door_api_url=http://127.0.0.1:3001/api
```
Comment (tochev) [inline door_ivr/door_ivr.test.conf]: Currently the fauna mock is on 3001 - maybe move the mock to 3002 and leave 3002 in the tests?

**Review by tochev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (user890104) [reply README.md:22]: I'll keep that in mind.

File: README.md (hunk 2 above)
Comment (user890104) [reply README.md:3]: OK

File: door_ivr/door_ivr.test.conf (hunk 3 above)
Comment (user890104) [reply door_ivr/door_ivr.test.conf]: Sounds like a plan

**Pull request merged**